script.js

```diff
@@ -0,0 +1,159 @@
+
+// Global constants
+let APIpage = 1;
+
+/////////////////////////////
+///// DISPLAY MOVIES ///////
+/////////////////////////////
+
+// function gets movies from API
+function getMovies(){
+    const options = {
+        method: 'GET',
+        headers: {
+          accept: 'application/json',
+          Authorization: 'Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIyNzZlZWIzODJmZmNiMjMyNTFiMjJkNzRlNzg1OTQzMyIsInN1YiI6IjY0ODAwNWRlNjQ3NjU0MDE0MzMyZjQyNiIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.tZPmNIXblH37hDQ-EAzomK8hpXr3j9MrxXfEOik9Pms'
+        }
+      };
+      // add page counter
+      fetch('https://api.themoviedb.org/3/movie/now_playing?language=en-US&page=' + APIpage.toString(), options)
+        .then(response => response.json())
+        .then((data) => {
+            console.log(data)
+            for (let i = 0; i < data.results.length; i++) {
+                generateCards(data.results[i]);
+            }})
+}
+
+
+function generateCards(movieObject){
+    // create star
+    let star = document.createElement('span');
+    star.classList.add('star');
+    let starContent = document.createTextNode('⭐️');
+    star.appendChild(starContent);
+
```

Comment:
recommend using a helper function here to make code more DRY; since we create and append multiple things